cms/djangoapps/contentstore/views/course.py

```diff
@@ -1422,6 +1453,18 @@ def group_configurations_detail_handler(request, course_key_string, group_config
             return JsonResponse(status=204)
 
 
+def has_course_enabled_content_experiments(course):
+    """
+    Returns True if Studio should show the "Experiment Group
```

Comment:
I thought the docstring does match... this method returns whether or not we should show the "Experiment Group Configurations" section on the group configurations page, right?